Corona-Warn-App/src/main/res/layout/fragment_information.xml

```diff
@@ -29,14 +29,16 @@
             app:layout_constraintStart_toStartOf="parent"
             app:layout_constraintTop_toBottomOf="@+id/information_header">
 
-            <androidx.constraintlayout.widget.ConstraintLayout
+            <LinearLayout
                 android:layout_width="match_parent"
-                android:layout_height="wrap_content">
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/spacing_normal"
```

Comment:
Why did you add spacing here?

Reply:
I meant to move it from line 43 to here, and in the process duplicated it accidentally. I updated my PR to remove the other occurence.